Library/Homebrew/extend/os/mac/search.rb

```diff
@@ -48,7 +47,7 @@ def search_casks(string_or_regex)
           else
             cask.token
           end
-        end
+        end.uniq
```

Comment:
```suggestion
        end
```

`|=` will already ensure uniqueness.

Reply:
Without `uniq`, the search results will show duplicate cask names. Because `result`  will include the same cask's `full_name` and one's `cask_tokens`


https://github.com/Homebrew/brew/blob/0758631d3db05338f79aa675a963891eafcf50db/Library/Homebrew/extend/os/mac/search.rb#L43


without `uniq`
```bash
brew search dockertoolbox
==> Formulae
docker-ls

==> Casks
docker-toolbox docker-toolbox
```

with `uniq`
```bash
brew search dockertoolbox
==> Formulae
docker-ls

==> Casks
docker-toolbox
```

Reply:
I see, in that case, what about doing `+=` above given we're doing `uniq` here anyway?

Reply:
@MikeMcQuaid 
Could you review this PR?